.github/workflows/generate-js.yml

```diff
@@ -0,0 +1,50 @@
+name: 'Generate JavaScript code and open pull request with it'
+
+on:
+  push:
+    branches-ignore:
+     - 'renovate/*'
+     - 'github-actions/generate-js/*'
+    paths-ignore:
```

Comment:
Could just omit this, we can rely on the diff.

Reply:
I just wanted to avoid unneeded work for github actions, but sure, will do